recipes/server.rb

```diff
@@ -22,4 +22,8 @@
 include_recipe 'lvm'
 include_recipe 'gluster::server_setup'
 include_recipe 'gluster::server_extend'
-include_recipe 'gluster::volume_extend'
+include_recipe 'gluster::volume_extend' if begin
+                                             Gem::Specification.find_by_name('di-ruby-lvm')
+                                           rescue
+                                             Gem::LoadError
```

Comment:
It looks like this will still get passed up and cause the run to bomb out.  I think you mean to have `Gem::LoadError` on the same like as the `rescue` statement

```
[2] pry(main)> puts 'pass' if begin
[2] pry(main)*   Gem::Specification.find_by_name('di-ruby-lvm')              
[2] pry(main)* rescue              
[2] pry(main)*   Gem::LoadError              
[2] pry(main)* end              
Gem::LoadError: Could not find 'di-ruby-lvm' (>= 0) among 179 total gem(s)
Checked in 'GEM_PATH=/Users/username/.gem/ruby/2.2.0:/usr/local/var/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0', execute `gem env` for more information
from /usr/local/var/rbenv/versions/2.2.2/lib/ruby/2.2.0/rubygems/dependency.rb:315:in `to_specs'
[3] pry(main)>
```